extensions/amp-nexxtv-player/0.1/test/test-amp-nexxtv-player.js

```diff
@@ -0,0 +1,127 @@
+/**
```

Comment:
After you verify that `autoplay` works fine manually, please add your players to `test-video-players.js` and include the script in `video-players.html` and run `gulp test --integration` to run the test suite. These tests ensure the video-interface is implemented correctly by black-box testing it. This ensures methods such as `play`, `pause`, `mute`, etc.. actually work and the events actually propagate back from the iframed player.